scripts/package/postinstall.sh

```diff
@@ -16,6 +16,20 @@
 
 set -e
 
+BDOT_CONFIG_HOME="/opt/observiq-otel-collector"
+
+install() {
+    mkdir -p "${BDOT_CONFIG_HOME}"
+    chmod 0755 "${BDOT_CONFIG_HOME}"
+    chown observiq-otel-collector:observiq-otel-collector "${BDOT_CONFIG_HOME}"
+    rm -f "${BDOT_CONFIG_HOME}/observiq-otel-collector" || true
```

Comment:
This is required because binary files cannot be overwritten when they are running within a process. It is safe to `rm` them, however. The underlying contents / inode is preserved until the process stops running.

The `cp` command will place the new binary.